package.json

```diff
@@ -93,7 +93,7 @@
     "eth-json-rpc-errors": "^1.1.0",
     "eth-json-rpc-filters": "^4.1.1",
     "eth-json-rpc-infura": "^4.0.1",
-    "eth-json-rpc-middleware": "^4.2.0",
+    "eth-json-rpc-middleware": "rekmarks/eth-json-rpc-middleware#update-rpc-engine",
```

Comment:
This needs to be merged into the main repo and published.

Reply:
This one we can actually leave be; I was getting paranoid about package resolutions in `yarn.lock` not working while testing.